### ISSUE TITLE
Errors using terraform v0.15.0 (deprecation removals)

### ISSUE THREAD

**Issue by andrmoel** [dealmore/terraform-aws-next-js#115]
Using terraform version 0.15.0 is causing several errors

```
│ Error: Error in function call
│ 
│   on .terraform/modules/tf_next.proxy.edge_proxy/iam.tf line 25, in data "aws_iam_policy_document" "assume_role":
│   25:       identifiers = distinct(concat(slice(list("lambda.amazonaws.com", "edgelambda.amazonaws.com"), 0, var.lambda_at_edge ? 2 : 1), var.trusted_entities))
│ 
│ Call to function "list" failed: the "list" function was deprecated in Terraform v0.12 and is no longer available; use tolist([ ... ]) syntax to write a literal list.

│ Error: Error in function call
│ 
│   on .terraform/modules/tf_next.proxy.edge_proxy/outputs.tf line 98, in output "s3_object":
│   98:   value       = map("bucket", local.s3_bucket, "key", local.s3_key, "version_id", local.s3_object_version)
│     ├────────────────
│     │ local.s3_bucket is a string, known only after apply
│     │ local.s3_key will be known only after apply
│     │ local.s3_object_version will be known only after apply
│ 
│ Call to function "map" failed: the "map" function was deprecated in Terraform v0.12 and is no longer available; use tomap({ ... }) syntax to write a literal map.

```

**Comment by ofhouse** [comment]
Ah, haven't switched to 0.15 yet, will push out a fix for this later today.
Thanks for reporting 👍

**Comment by ofhouse** [comment]
This took a day longer because we also had to release a new version of the the [image optimizer module](https://github.com/dealmore/terraform-aws-next-js-image-optimization).

Compatibility issues with Terraform 0.15 are now fixed with the [`v0.8.1` release](https://github.com/dealmore/terraform-aws-next-js/releases/tag/v0.8.1).
